sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -193,4 +195,33 @@ export class ApplicationApi extends HttpBaseClient {
       null,
     );
   }
+
+  /**
+   * Triggers a manual reassessment for the application.
```

Comment:
Can the comment on sources/packages/web/src/services/ApplicationService.ts be aligned with this one?